pkg/sass_api/pubspec.yaml

```diff
@@ -2,15 +2,15 @@ name: sass_api
 # Note: Every time we add a new Sass AST node, we need to bump the *major*
 # version because it's a breaking change for anyone who's implementing the
 # visitor interface(s).
-version: 14.2.0
+version: 14.2.1
```

Comment:
This is adding a new feature to `sass_api`, so it should be `14.3.0`

Reply:
Whoops. Had it as a minor version originally, but accidentally did a patch version when I re-did it after syncing